packages/ui/components/editor/stylesEditor.css

```diff
@@ -33,7 +33,6 @@
   border-bottom-left-radius: 6px;
   border-bottom-right-radius: 6px;
   overflow: scroll;
-  resize: vertical;
```

Comment:
why did you remove this?

Reply:
See, I don't think there is a need to increase the min-height for the editor because it automatically gets bigger when the content gets longer.

However, if a user uses the resize controller to resize the height of the editor. It will then stop adjusting to the size of the content. It will no longer adapt to the content length but if we remove the option to resize the editor it will always adapt to the content.

But, I'm not aware of any other place where the editor size might need to be adjusted by the user (I don't think it would be a necessity). So if that's the case then please ignore this change.